packages/app/components/Post/Post.tsx

```diff
@@ -73,11 +75,9 @@ const Post = ({
 }: PostProps) => {
   const { address, isAuthenticated } = useWallet();
   const queryClient = useQueryClient();
-
-  const {
-    sort,
-    siwe: { address: siweAddress },
-  } = useStore();
+  const { sort } = useStore();
+  const provider = useProvider();
+  const [resolvedName] = useEnsLookup([postedBy], provider);
```

Comment:
maybe i should use a method that reads from the store. this potentially could cause an additional lookup, though I wanted to be sure anywhere `<Post />` was used, the address was looked up.